### PR TITLE
Ft routine

### PR DESCRIPTION
**Pull request by dreplica**
Screenshot_20200501-221348_Expo

#### What does this PR do?

it displays daily routine for a clients Regime

#### Description of Task to be completed?

Have the Routine screen show the following:

1. A header that displays Routine and a subHeader that displays recommended.
2. Displays seperates cards, with Routines.
3. each card has an image, time, title and description.

#### How should this be manually tested?

After cloning the repo,
`CD` into it and `RUN`

javascript
git checkout develop

git pull origin develop

yarn install

yarn test

Using an emulator: Launch the app, and you should see the home screen after loading.

### CODE REVIEW

**Comment by dreplica** [comment]
this is the loaded screen

Screenshot_20200501-221355_Expo

**Comment by dreplica** [comment]
this is the loaded screen 
Screenshot_20200501-223246_Expo

File: src/navigator/BottomNavigator.tsx [hunk 1]
```diff
@@ -7,6 +7,7 @@ import HairCareIcon from '../../assets/icons/hair_care';
 import SettingIcon from '../../assets/icons/settings';
 import TabBarLabel from '../commons/tab-bar-label';
 import { IconContainer } from './styles';
+import RoutineScreen from '../screens/regimen/RegimenScreen';
```
Comment (ogbodo) [inline src/navigator/BottomNavigator.tsx]: move this up and make styles the last import

File: src/screens/regimen/RegimenScreen/cards/Card.tsx [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+  content: string;
+  heading: string;
+}
+
+export default function Card(props: CardProps) {
+  const [order, setOrder] = useState<'row' | 'row-reverse'>('row');
+  const { fonts, colors } = useThemeContext();
+
+  useEffect(() => {
+    if (props.order === 1) {
```
Comment (ogbodo) [inline src/screens/regimen/RegimenScreen/cards/Card.tsx]: move this logic of determining position of the image to the parent of this card. Let this card be responsible to only display data not making a decision that causes re-rendering each time its been used.

**Review by ogbodo — CHANGES_REQUESTED** [review]

File: src/navigator/BottomNavigator.tsx (hunk 1 above)
Comment (dreplica) [reply src/navigator/BottomNavigator.tsx]: Okay, I removed it totally, I was using it to see my screen

File: src/screens/regimen/RegimenScreen/cards/Card.tsx (hunk 2 above)
Comment (dreplica) [reply src/screens/regimen/RegimenScreen/cards/Card.tsx]: Please check the current one if it fits better

**Review by ogbodo — APPROVED** [review]

**Pull request closed without merging**